core-web/apps/dotcms-ui/src/app/view/components/dot-content-compare/store/dot-content-compare.store.ts

```diff
@@ -87,7 +87,28 @@ export class DotContentCompareStore extends ComponentStore<DotContentCompareStat
                                         return { contentType, contents };
                                     })
                                 );
-                        })
+                        }),
+                        switchMap(
+                            (value: {
+                                contentType: DotCMSContentType;
+                                contents: DotCMSContentlet[];
+                            }) => {
+                                return !value.contents.some(
+                                    (content) => content.inode === data.inode
+                                )
+                                    ? this.dotContentletService
```

Comment:
What happed if any of the requests fail?

Reply:
Thanks for the feedback, I try to improve it here https://github.com/dotCMS/core/pull/27663/commits/2349909b1927804cd165c7d53821e2ffafe66e10